tajo-cli/src/main/java/org/apache/tajo/cli/tools/TajoDump.java

```diff
@@ -216,6 +217,8 @@ private static void dumpDatabase(TajoClient client, String databaseName, PrintWr
         System.err.println("ERROR:" + tableName + "," + e.getMessage());
       }
     }
+
+    LOG.warn("Partition dump and restore are not supported yet");
```

Comment:
Where is this message printed? Have you ever tested by yourself?
In addition, this message should be printed only when users try to dump partitioned table.